bootstrap/kubeadm/config/rbac/auth_proxy_service.yaml

```diff
@@ -1,6 +1,10 @@
 apiVersion: v1
 kind: Service
 metadata:
+  annotations:
```

Comment:
should these changes also be applied in each infra provider?

Reply:
Yes. In `capo` the annotation is already set
https://github.com/kubernetes-sigs/cluster-api-provider-openstack/blob/master/config/rbac/auth_proxy_service.yaml#L4

Reply:
Will go through the other infra provider and add the missing annotation.

Reply:
Hi @CecileRobertMichon 

just created three PRs for providers under `kubernetes-sigs`-Org, because CLA already signed. Have to check the legal stuff on the remaining others first. Hope this doesn't block this PR ;-)

Summary:
- new PRs:
  - [Packet](https://github.com/kubernetes-sigs/cluster-api-provider-packet/pull/231)
  - [IBM Cloud](https://github.com/kubernetes-sigs/cluster-api-provider-ibmcloud/pull/310)
  - [DigitalOcean](https://github.com/kubernetes-sigs/cluster-api-provider-digitalocean/pull/240)
- `annotation` already exist:
  - [AWS](https://github.com/kubernetes-sigs/cluster-api-provider-aws/blob/master/config/rbac/auth_proxy_service.yaml)
  - [Metal3](https://github.com/metal3-io/cluster-api-provider-metal3/blob/master/config/rbac/auth_proxy_service.yaml)
  - [GCP](https://github.com/kubernetes-sigs/cluster-api-provider-gcp/blob/master/config/rbac/auth_proxy_service.yaml)
  - [OpenStack](https://github.com/kubernetes-sigs/cluster-api-provider-openstack/blob/master/config/rbac/auth_proxy_service.yaml)
  - [vSphere](https://github.com/kubernetes-sigs/cluster-api-provider-vsphere/blob/master/config/rbac/auth_proxy_service.yaml)

Reply:
what about [Azure](https://github.com/kubernetes-sigs/cluster-api-provider-azure)?

Reply:
annotation was already there but was removed last year. 

I created [Issue #1222](https://github.com/kubernetes-sigs/cluster-api-provider-azure/issues/1222) to get in contact with Azure team.

Reply:
Azure PR [#1320](https://github.com/kubernetes-sigs/cluster-api-provider-azure/pull/1320) created